src/tasks/minions/minigames/tobActivity.ts

```diff
@@ -141,6 +141,11 @@ export const tobTask: MinionTask = {
 				// Refund initial 100k entry cost
 				userLoot.add('Coins', 100_000);
 
+				// OSB only: Remove elite clue scroll if user has one in bank
+				if (user.owns('Clue scroll (elite)')) {
+					userLoot.remove('Clue scroll (elite)', 1);
+				}
```

Comment:
Fine as long as its not possible to get 3+

Reply:
Correct, this is after the loot sim from tob.ts so at this point the users loot should only have max 1 elite clue scroll. Then if they have one in the bank it will delete it.